.github/workflows/release.yml

```diff
@@ -12,7 +12,7 @@ permissions:
 jobs:
   build:
     name: Release
-    runs-on: ubuntu-latest
+    runs-on: macos-13
```

Comment:
Could we just use mac to do codesign or just use mac to build mac release but not others?

Reply:
Updated to build on linux and codesign on macOS.